composer/spec/dependabot/composer/file_fetcher_spec.rb

```diff
@@ -59,6 +59,13 @@
       to match_array(%w(composer.json composer.lock))
   end
 
+  it "provides the composer version" do
+    expect(file_fetcher_instance.package_manager_version).to eq({
+      ecosystem: "composer",
+      package_managers: { "composer" => "2" }
```

Comment:
If we return the full version, this test will complain, which will remind us to truncate the patch version rather than fully reporting it.